components/widgets/menu/menu_items/menu_item_action.tsx

```diff
@@ -8,6 +8,8 @@ import menuItem from './menu_item';
 
 type Props = {
     onClick: (e: React.MouseEvent) => void;
+    onMouseEnter?: (e: React.MouseEvent) => void;
```

Comment:
you can revert all of this too ... not needed anymore ... thanks!

Reply:
Done.